### ISSUE TITLE
ergotop is broken

### ISSUE THREAD

**Issue by dselman** [accordproject/ergo#591]
```
npm install -g @accordproject/ergo-cli
/Users/dselman/.nvm/versions/node/v8.15.0/bin/ergoc -> /Users/dselman/.nvm/versions/node/v8.15.0/lib/node_modules/@accordproject/ergo-cli/bin/ergoc.js
/Users/dselman/.nvm/versions/node/v8.15.0/bin/ergorun -> /Users/dselman/.nvm/versions/node/v8.15.0/lib/node_modules/@accordproject/ergo-cli/bin/ergorun.js
/Users/dselman/.nvm/versions/node/v8.15.0/bin/ergotop -> /Users/dselman/.nvm/versions/node/v8.15.0/lib/node_modules/@accordproject/ergo-cli/bin/ergotop.js
+ @accordproject/ergo-cli@0.7.3
added 144 packages from 87 contributors in 11.061s
Dan-MacBook-Pro:cicero-core dselman$ ergotop
module.js:550
    throw err;
    ^

Error: Cannot find module 'mkdirp'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/dselman/.nvm/versions/node/v8.15.0/lib/node_modules/@accordproject/ergo-cli/node_modules/composer-concerto/lib/codegen/filewriter.js:18:16)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
Dan-MacBook-Pro:cicero-core dselman$ 
```

**Comment by jeromesimeon** [comment]
I think this might be a bug in composer-concerto. `mkdirp` should be in the dependencies, not the devDependencies.

**Comment by jeromesimeon** [comment]
This affects the other Ergo CLI tools:
```
bash-3.2$ ergoc --version
module.js:550
    throw err;
    ^

Error: Cannot find module 'mkdirp'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/node_modules/composer-concerto/lib/codegen/filewriter.js:18:16)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

**Comment by jeromesimeon** [comment]
Fixed in dev:
```
bash-3.2$ npm install -g @accordproject/ergo-cli@0.7.4-20190329204625 
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/ergoc -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/bin/ergoc.js
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/ergorun -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/bin/ergorun.js
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/ergotop -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/bin/ergotop.js
+ @accordproject/ergo-cli@0.7.4-20190329204625
added 38 packages from 8 contributors, removed 18 packages, updated 28 packages and moved 1 package in 28.852s
bash-3.2$ ergoc --help
Usage: ergoc [options] [cto files] [ergo files]
  --version       Print version and exit
  --target <lang> Target platform (default: es6) (available: es5,es6,cicero,java)
  --link          Adds the Ergo runtime to the target code (es5,es6,cicero only)
  --monitor       Produce compilation time information
  -help           Display this list of options
  --help          Display this list of options
```

**Comment by jeromesimeon** [comment]
Fixed in 0.8:
```
bash-3.2$ npm install -g @accordproject/ergo-cli@0.8.0 
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/ergorun -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/bin/ergorun.js
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/ergoc -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/bin/ergoc.js
/Users/jeromesimeon/.nvm/versions/node/v8.12.0/bin/ergotop -> /Users/jeromesimeon/.nvm/versions/node/v8.12.0/lib/node_modules/@accordproject/ergo-cli/bin/ergotop.js
+ @accordproject/ergo-cli@0.8.0
updated 1 package in 7.91s
bash-3.2$ ergoc --version 
The Ergo compiler, version 0.8.0
bash-3.2$ ergoc --help
Usage: ergoc [options] [cto files] [ergo files]
  --version       Print version and exit
  --target <lang> Target platform (default: es6) (available: es5,es6,cicero,java)
  --link          Adds the Ergo runtime to the target code (es5,es6,cicero only)
  --monitor       Produce compilation time information
  -help           Display this list of options
  --help          Display this list of options
bash-3.2$ ergotop
Welcome to ERGOTOP version 0.8.0
ergo$ return 41 + 1
Response. 42 : Integer
ergo$ 
```
